### PR TITLE
linting and route to get genome by taxID.

### PR DESCRIPTION
**Pull request by daviortega**
copied from /$genomes directory without the /genes route

### CODE REVIEW

File: mist-api/src/lib/RouteHelper.js [hunk 1]
```diff
@@ -90,15 +90,15 @@ class RouteHelper {
...
 			this.model_.findOne(criteria)
-			.then((entity) => {
-				if (!entity) {
-					next(errors.notFoundError)
-					return
-				}
-
-				res.json(entity)
-			})
-			.catch(next)
```
Comment (lukeulrich) [inline mist-api/src/lib/RouteHelper.js]: This is a style concern and given that all our current formatting follows the indent pattern above, please do not change this. If we elect as a team to change it at a later time, then we should make a single PR dedicated to fixing that across the project.

File: mist-api/src/routes/genome_by_taxid/get.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'use strict'
```
Comment (lukeulrich) [inline mist-api/src/routes/genome_by_taxid/get.js]: Exact replica of `routes/genomes/get.js`. Remove this file.

File: mist-api/src/lib/RouteHelper.js (hunk 1 above)
Comment (daviortega) [reply mist-api/src/lib/RouteHelper.js]: this was pointed out by the eslinter config by our team... I just fixed because it was telling me to :)

Comment (lukeulrich) [reply mist-api/src/lib/RouteHelper.js]: K. Will have to take a look at the linter rules.

File: mist-api/src/routes/genome_by_taxid/get.js (hunk 2 above)
Comment (daviortega) [reply mist-api/src/routes/genome_by_taxid/get.js]: well, if we will work on `parseCriteriaForMany`, then we should just reject this branch altogether and rework the code to support simple query matching. I am going to close it.

**Pull request closed without merging**